content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -522,24 +522,33 @@ This means that support is still available but will be removed in a future Kuber
 * [`rbd`](/docs/concepts/storage/volumes/#rbd) - Rados Block Device (RBD) volume
   (**deprecated** in v1.28)
 
-Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
+The following PersistentVolume types are supported through the `CSIMigration` feature gate and require the corresponding CSI driver to be installed in the cluster. Kubernetes offers built-in APIs to facilitate their use.
 
 * [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
-  (**not available** in v1.27)
+  &mdash; but for Kubernetes version 1.27 and onwards, this volume type is **only** available via a separate CSI driver 
```

Comment:
AWS EBS CSI migration went [GA in 1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md#feature-14)

Reply:
No, the CSI driver was really mandatory in 1.25. A Kubernetes cluster is not able to provision/delete or mount/unmount in-tree `awsElasticBlockStore` volumes without it. The only exception is attaching and detaching them to/from kubelets < 1.25, that's why we removed it in 1.27.

This applies to all plugins below.

Reply:
Updated the version in the latest commit.